### PR TITLE
Space.wizard.to.select.space

### PR DESCRIPTION
**Pull request by corinnekrych**
WIP waiting for https://github.com/almighty/almighty-core/pull/1148 to be merged fix

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f4a35eafc7464be18fbbb1827f2ff447782ef16d
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/app/create/codebases/services/github.service.mock.ts [hunk 1]
```diff
@@ -73,114 +87,114 @@ export class ContextsMock {
...
-  "forks": 44,
-  "open_issues": 250,
-  "watchers": 25,
-  "default_branch": "master",
-}
+  {
+    "id": 54260706,
+    "name": "almighty-core",
+    "full_name": "almighty/almighty-core",
+    "owner": {
```
Comment (kwk) [inline src/app/create/codebases/services/github.service.mock.ts]: Are those changes really needed here?

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f4a35eafc7464be18fbbb1827f2ff447782ef16d
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 71a7f162ccab604696c3ee72c69a4c30414aa301
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by joshuawilson** [comment]
[test]

**Comment by corinnekrych** [comment]
@joshuawilson unit test is failing because we need the ngx-fabric8-wit from this PR https://github.com/fabric8-ui/ngx-fabric8-wit/pull/101
I've tested it locally with "npm link"

**Comment by jfchevrette** [comment]
[test]

**Comment by rawlingsj** [comment]
This PR CI build is failing with this error also it needs conflicts resolving.
```
Error in bail mode: [at-loader] ./src/app/app.module.ts:125:3 
    TS2305: Module '"/home/jenkins/workspace/auto_fabric8-ui_PR-1424-Y6PRCEK6XSAHBW4ES26AAZZCZ5S7OEN62KSJ4YFYBGCRDGLBJ2NA@2/node_modules/ngx-fabric8-wit/index"' has no exported member 'SpaceTemplateService'.
```

**Comment by joshuawilson** [comment]
Is this still blocked? I think this was one of the train goals for planner. Is this PR still relevant? If so, please rebase it.

**Comment by joshuawilson** [comment]
I think the work was done in #2664, closing. Reopen if this is still need.

**Pull request closed without merging**